### PR TITLE
Add CBT suite to be run for every check-in to stork

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rkulkarni@purestorage.com>

**What type of PR is this?**
integration-test

**What this PR does / why we need it**:

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
2.11

### CODE REVIEW

**Comment by lpitstickpstg** [comment]
How much time does this add?

**Comment by Rohit-PX** [comment]
Tested the suite here: https://jenkins.pwx.dev.purestorage.com/job/Stork/view/Stork%202.11-dev%20%20C7/job/stork-2.11-px-2.11-dev-migr/129/console

**Comment by adityadani** [comment]
1hr 17 mins is not too bad.

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -126,6 +126,25 @@ func TestSnapshot(t *testing.T) {
 	t.Run("testWebhook", testWebhook)
 }
 
+func TestStorkCbt(t *testing.T) {
+	t.Run("deploymentTest", deploymentMigrationTest)
+	t.Run("deploymentMigrationReverseTest", deploymentMigrationReverseTest)
```
Comment (adityadani) [inline test/integration_test/common_test.go:131]: Can we use failoverAndFailback test instead? It is a superset of the deploymentMigrationReverseTest

**Comment by Rohit-PX** [comment]
> How much time does this add?

This runs for approx 1hr 15 min (75 min) and will be triggered after every check-in (on branches that we want).

File: test/integration_test/common_test.go (hunk 1 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Done.

Comment (Rohit-PX) [reply test/integration_test/common_test.go]: @adityadani added vendor changes as discussed.

**Review by adityadani — APPROVED** [review]

**Pull request merged**